packages/atomic/cypress/e2e/search-box/search-box.cypress.ts

```diff
@@ -146,6 +171,110 @@ describe('Search Box Test Suites', () => {
         );
       });
 
+      describe('with custom suggestions provider', () => {
```

Comment:
Most of the test setup complexity is because here we are not testing an atomic component per say, but some arbitrary code that register/hooks into the custom search box suggestion system.